### PR TITLE
[Enhancement] Task support priority for manual refresh materialized view

### LINKED ISSUE #7898: Task support priority
## Enhancement
Manual refresh usually has a higher priority than automatic refresh in materialized view scenarios. In order to support this scenario, Task support priority is required.

### PR DESCRIPTION
**Pull request by Astralidea**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #7898

## Problem Summary(Required) ：

Set the priority of manually refreshing the materialized view to NORMAL(50), and the default of other types is LOWEST(0).

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 15 / 18 (83.33%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/execution/RefreshMaterializedViewExecutor.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/connector/ConnectorMetadata.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskManager.java|2|3|66.67%|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskRunManager.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskRun.java|4|4|100.00%|

File: fe/fe-core/src/main/java/com/starrocks/scheduler/Constants.java [hunk 1]
```diff
@@ -34,4 +34,6 @@ public enum TaskRunState {
         SUCCESS,
     }
 
+    public static final int DEFAULT_TASK_RUN_PRIORITY = 0;
+    public static final int MANUAL_TASK_RUN_PRIORITY = 20;
```
Comment (kangkaisen) [inline fe/fe-core/src/main/java/com/starrocks/scheduler/Constants.java:38]: prefer using a enum, not respective constant value

Comment (imay) [reply fe/fe-core/src/main/java/com/starrocks/scheduler/Constants.java:38]: priory should be integer and in java enum is not easy to convert to each other

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 21 / 24 (87.50%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/execution/RefreshMaterializedViewExecutor.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/connector/ConnectorMetadata.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskManager.java|2|3|66.67%|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskRunManager.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/Constants.java|6|6|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskRun.java|4|4|100.00%|

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage check]

:heart_eyes: **pass** : 21 / 24 (87.50%)




#### file detail

|   |path|covered line|new line|coverage|
|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/execution/RefreshMaterializedViewExecutor.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/connector/ConnectorMetadata.java|0|1|00.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskManager.java|2|3|66.67%|
|:large_blue_circle:|com/starrocks/scheduler/persist/TaskRunStatus.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|1|1|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskRunManager.java|4|4|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/Constants.java|6|6|100.00%|
|:large_blue_circle:|com/starrocks/scheduler/TaskRun.java|4|4|100.00%|

**Review by kangkaisen — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 2]
```diff
@@ -2985,7 +2985,7 @@ public void alterMaterializedView(AlterMaterializedViewStatement stmt) throws Dd
     }
 
     @Override
-    public void refreshMaterializedView(String dbName, String mvName) throws DdlException, MetaNotFoundException {
+    public void refreshMaterializedView(String dbName, String mvName, int priority) throws DdlException, MetaNotFoundException {
```
Comment (ABingHuang) [inline fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:2988]: why not use TaskRunPriority as argument?

Comment (Astralidea) [reply fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java:2988]: The priority is not only those of the enumeration values, there may be more complicated situations later, so it should be of the int type, providing the enumeration value is just a simple template.

**Review by ABingHuang — APPROVED** [review]
LGTM

**Review by gengjun-git — APPROVED** [review]

**Pull request merged**